rust/processor/src/db/postgres/schema.rs

```diff
@@ -837,8 +851,8 @@ diesel::table! {
         supply_aggregator_table_key_v1 -> Nullable<Text>,
         #[max_length = 10]
         token_standard -> Varchar,
-        is_token_v2 -> Nullable<Bool>,
         inserted_at -> Timestamp,
+        is_token_v2 -> Nullable<Bool>,
```

Comment:
This needed to be changed around otherwise whenever the migrations are re-ran, the ordering of the produced struct's constructor changes causing some issues.

Reply:
to add a bit of context: this is reverting the change I made manually. this order should be always the same unless we change in sql file.